### PR TITLE
Add eattheblocks tutorials

### PR DESCRIPTION
**Pull request by jklepatch**
Added links to eattheblocks tutorials in developers ressources

### CODE REVIEW

File: docs/developers/index.md [hunk 1]
```diff
@@ -439,6 +440,13 @@ The Ethereum network is made up of many nodes who run compatible client software
 
 ### General Learning {#general-learning}
 
+**EatTheBlocks -** **_Learn to build Ethereum Dapps & Solidity smart contracts with video tutorials
+
+- [Youtube - 1 new video / day](https://www.youtube.com/c/eattheblocks)
+- [Website](https://eattheblocks.com)
+
+- [ethereum.stackexchange.com](https://ethereum.stackexchange.com/)
```
Comment (samajammin) [inline docs/developers/index.md]: This looks to be a duplicate of line 452

File: docs/developers/index.md [hunk 2]
```diff
@@ -439,6 +440,13 @@ The Ethereum network is made up of many nodes who run compatible client software
 
 ### General Learning {#general-learning}
 
+**EatTheBlocks -** **_Learn to build Ethereum Dapps & Solidity smart contracts with video tutorials
```
Comment (CPSTL) [inline docs/developers/index.md]: Markdown error

File: docs/developers/index.md (hunk 1 above)
Comment (jklepatch) [reply docs/developers/index.md]: fixed :)

**Comment by jklepatch** [comment]
@samajammin thanks for your comments I have addressed them :)
As for my links being on top of each section, well its better than being at the bottom haha!
But more seriously I keep my tutorials very up-to-date and I upload new tutorial videos for ethereum developers every day on my channel EatTheBlocks.

**Comment by jklepatch** [comment]
@samajammin @CPSTL I have made the requested changes, could you guys have a last look and merge if everything looks good? :) thx

**Comment by jklepatch** [comment]
@samajammin @CPSTL any updates on this?

**Comment by faraggi** [comment]
In this case, I think @jklepatch's videos being on top in the sections he added is coherent, for his content is more up to date than the previous top listed content. 

But we should figure out a way to order content, maybe alphabetically?

**Comment by CPSTL** [comment]
> @samajammin @CPSTL any updates on this?

@jklepatch Until we have a way to order the content, we typically ask users to add their content to the bottom of each section / list.

**Comment by samajammin** [comment]
@jklepatch - sorry for the delay on this. I agree with @faraggi that a discussion on ordering would be beneficial, perhaps with some way to prioritize the "freshest" content but for now, as @CPSTL mentioned, we'd like folks to add new additions to the bottom of each list to keep treatment consistent. Please make that update & we'll get this merged ASAP. Thanks!

**Comment by samajammin** [comment]
@jklepatch checking in. Go ahead & make those updates & we'll get this merged. Thanks!

**Comment by jklepatch** [comment]
@samajammin just did :)

**Comment by CPSTL** [comment]
@all-contributors please add @jklepatch for content

**Pull request merged**